src/node.cc

```diff
@@ -800,6 +800,14 @@ Local<Value> GetDomainProperty(Environment* env, Local<Object> object) {
 void DomainEnter(Environment* env, Local<Object> object) {
   Local<Value> domain_v = GetDomainProperty(env, object);
   if (domain_v->IsObject()) {
+    if (env->EmitDomainWarning()) {
+      ProcessEmitDeprecationWarning(env,
+        "Using a domain property in MakeCallback is deprecated. Use the "
+        "async_context variant of MakeCallback or the AsyncResource class "
+        "instead.",
+        "DEPXXXX");
```

Comment:
4 space indent for line continuations and `env` should arguably go on a line of its own.